libraries/AP_GPS/AP_GPS_NMEA.cpp

```diff
@@ -361,7 +429,21 @@ bool AP_GPS_NMEA::_term_complete()
         case _GPS_SENTENCE_GPRMC + 2: // validity (RMC)
             _gps_data_good = _term[0] == 'A';
             break;
+        case _GPS_SENTENCE_GPRMC + 12: // Mode indicator (RMC)
+            /* A = Autonomous mode
+             * D = Differential mode
+             * F = Float RTK  (specific for NV08C (not official NMEA 0813))
+             * R = Real Time Kinematic (specific for NV08C (not official NMEA 0813))
+             * E = Estimated (dead reckoning) mode
+             * M = Manual input mode
+             * S = Simulator mode
+             * N = Data not valid
+             */
```

Comment:
It seems it was added in version 2.3: http://www.gpsinformation.org/dale/nmea.htm#2.3
But the _gps_data_good below is wrong, the E shouldn't be there according to the protocol. In reality I'm not sure we should even be setting _gps_data_good here, it is already checked before.